### ISSUE TITLE
Balance Druid - Starsurge 

### ISSUE THREAD

**Issue by avaksx** [Valhalla-Gaming/Tracker#8265]
**Describe the issue you're having**: I have 40 Astral Power and I cannot cast Starsurge even tho I have enough Astral Power on me.

**Explain how you expect it work**:Fix.

**Steps to reproduce the problem**:

1. Check.
2. Test.
3. Fix.

**Links to Wowhead, YouTube, etc**:
http://www.wowhead.com/spell=78674/starsurge

![wowscrnshot_122616_152105](https://cloud.githubusercontent.com/assets/22096253/21483337/ba98fd7a-cb80-11e6-9a19-06005078f9e0.jpg)

**Comment by Milofajjour** [comment]
This happens when u end your last combat with extra astral power you can't use them ehen you exist the combst and enter a new one.

**Comment by avaksx** [comment]
No, that is totally different issue. I was on 0 Astral Power before starting to fight this npc.

I started to attack him and build up my AP and as you can see the following from the picture.
